participants.md

```diff
@@ -1 +1 @@
-sajal
+Gagan
```

Comment:
The description says that you need to add your name, and not replace mine. Please update the same.

Reply:
apologies, on it!